try.html

```diff
@@ -830,10 +830,14 @@ <h3 id="miscellaneous"> Miscellaneous </h3>
     <td><img src='/gitter/room/nwjs/nw.js.svg' alt=''/></td>
     <td><code>https://img.shields.io/gitter/room/nwjs/nw.js.svg</code></td>
   </tr>
-  <tr><th> JIRA issue: </th>
-    <td><img src='/jira/issue/https/issues.apache.org/jira/KAFKA-2896.svg' alt=''/></td>
+  <tr><th data-keywords='jira-issues'> JIRA issue: </th>
+    <td><img src='/jira/issue/https/issues.apache.org/jira/KAFKA-2896.svg?maxAge=2592000' alt=''/></td>
     <td><code>https://img.shields.io/jira/issue/https/issues.apache.org/jira/KAFKA-2896.svg</code></td>
   </tr>
+  <tr><th data-keywords='jira-sprint-completion'> JIRA sprint completion: </th>
+    <td><img src='/jira/sprint/https/issues.apache.org/jira/247.svg?maxAge=2592000' alt=''/></td>
+    <td><code>https://img.shields.io/jira/sprint/https/issues.apache.org/jira/247.svg</code></td>
```

Comment:
Could you document how to obtain this number, `247`? We have a `<div id=documentation>` in `try.html` in which you can add a div which you can refer to by its id through a `<th data-doc=…>` attribute.